install_scripts/templates/common/kubernetes.sh

```diff
@@ -224,6 +224,7 @@ EOF
 
     pushd archives
         rpm --upgrade --force *.rpm
+        service docker restart
```

Comment:
The docker binary on the host isn't found after this command. Restarting docker fixes it.